### ISSUE TITLE
Unable to stretch player view on the entire screen (Please make add an option to set webView.scrollView.contentInsetAdjustmentBehavior = .never)

### ISSUE THREAD

**Issue by erank3** [SvenTiigi/YouTubePlayerKit#28]
I am trying to use the player as a full screen -  inside a collection view (same experience as YouTube Kids)

however I can't get the video to stretch on the entire view size because webView.scrollView adjusts the view to fit the safeAreaInsets: 

<WKScrollView: 0x11f00d200; baseClass = UIScrollView; frame = (0 0; 812 375); clipsToBounds = YES; gestureRecognizers = <NSArray: 0x600001a15710>; layer = <CALayer: 0x6000014fcaa0>; contentOffset: {-50, 0}; contentSize: {712, 354}; adjustedContentInset: {0, 50, 21, 50}>

so the player view shifts.

as a workaround I had to expose webView and call:
            youTubePlayerViewController.webView.scrollView.contentInsetAdjustmentBehavior = .never

so a possible solution would be making it public

**Comment by SvenTiigi** [comment]
Hi @erank3,

Please checkout the [`develop`](https://github.com/SvenTiigi/YouTubePlayerKit/tree/develop) branch which contains a new property named `automaticallyAdjustsContentInsets` on the `YouTubePlayer.Configuration` (https://github.com/SvenTiigi/YouTubePlayerKit/commit/fa70945f48780b2f78102c5905b6a65b2531b618)

You can set this property to `false` to disable the content inset adjustment behavior.

```swift
let youTubePlayer = YouTubePlayer(
    source: .video(id: "..."),
    configuration: .init(
        automaticallyAdjustsContentInsets: false
    )
)
```

Please let me know if this change works for you ✌️

**Comment by erank3** [comment]
That worked!
Thanks so much for the quick response.

**Comment by SvenTiigi** [comment]
The changes are now available with the latest version [`1.1.12`](https://github.com/SvenTiigi/YouTubePlayerKit/releases/tag/1.1.12)
